cupyx/scipy/sparse/compressed.py

```diff
@@ -893,82 +801,21 @@ def _minor_reduce(self, ufunc, axis, nonzero):
                 major_index.
 
         """
-
+        shape = self.shape[int(not(axis))]
```

Comment:
- I found `1 - axis` in `cupyx/scipy/sparse/data.py`.
- For readability, could you give a more detailed name?  How about `shape` -> `out_shape` and `value` -> `out`?

Reply:
Good catch :).
Thanks!